src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import duke.command.Command;
+import duke.io.Storage;
+import duke.io.TaskList;
+import duke.parser.Parser;
```

Comment:
I like how you have structured your application! I think the classes are really well organized in their packages!